src/NzbDrone.Automation.Test/WantedPageTest.cs

```diff
@@ -0,0 +1,46 @@
+using System.Threading;
+using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Automation.Test.PageModel;
+using OpenQA.Selenium;
+
+namespace NzbDrone.Automation.Test
+{
+    [TestFixture]
+    public class WantedPageTest : AutomationTest
+    {
+        private PageBase page;
+
+        [SetUp]
+        public void Setup()
+        {
+            page = new PageBase(driver);
+        }
+
+        [Test]
+        public void hide_drone_update_by_default()
+        {
+            page.WantedNavIcon.Click();
+            page.WaitForNoSpinner();
+
+            Assert.Throws<NoSuchElementException>(() => page.Displayed(By.ClassName("icon-sonarr-refresh")));
+        }
+
+        [Test]
+        public void show_drone_update_if_drone_folder_set()
+        {
+            page.SettingNavIcon.Click();
+            page.WaitForNoSpinner();
+
+            page.FindByClass("x-download-client-tab").Click();
+            page.FindByClass("x-advanced-settings").Click();
+            page.Find(By.Name("downloadedEpisodesFolder")).SendKeys("C:\\");
```

Comment:
Is there a drone folder value I can use that lets this test pass cross-platform?

Reply:
There's an `.AsOsAgnostic()` extension method, but I don't think that'll work coz the validator checks if the folder exists. Better just create a temporary folder somewhere. The NzbDroneRunner instance has an AppData property that contains a temp folder just for that test. So create a subfolder there and use it as fake drone factory folder.
But don't spend too much time on those automation tests. Atm they only test whether the UI shows up and the main pages are browsable, little more than a smoketest.